applications_tests/lethe-fluid-block/cylinder_carreau_gd.prm

```diff
@@ -94,11 +94,13 @@ end
 #---------------------------------------------------
 
 subsection boundary conditions
-  set number = 3
+  set number = 4
   subsection bc 0
+    set   id = 0
```

Comment:
I don't think this is correctly indented. I see the same mistake in several parameter files. 
```suggestion
    set   id = 0
```

Reply:
I just reapplied prmindent to all files :)